src/redux/configureStore.js

```diff
@@ -1,12 +1,17 @@
 import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers } from 'redux';
+import thunk from 'redux-thunk';
 import categoriesReducer from './categories/categories';
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 but according to the Project requirements, the  `createAsyncThunk` must be used in the project for creating thunk action creators. Please address this 👍🏻 

```js
// This code is just an example,

import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```